helm/hpcc/templates/dali.yaml

```diff
@@ -1,5 +1,10 @@
 {{ range $.Values.dali -}}
 {{- if not .disabled -}}
+{{- $secretsCategories := list "system" }}
+{{- if or (has "data" .access) (has "dalidata" .access) -}}
```

Comment:
this won't work, because "access" is set as an internal variable, per dali sasha service.
See : https://github.com/hpcc-systems/HPCC-Platform/blob/c28c89ecd71992ebeb90218292c5caf2f9f18fd7/helm/hpcc/templates/dali.yaml#L40

Each sasha service container has different a access set as returned by getSashaServiceAccess, which is different from Dali's access requirements (which only requires system).

I think needs to be changed so done here just for Dali, and inside the range .services loop for the other containers.